<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>125</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>412</width>
    <height>125</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Weight (kg)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Tent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oxygen bottle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gas bottle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sleeping bag</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cauldron</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>0.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>Sleeping bag temperature (-1*C)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sleeping bag temperature (-5*C)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sleeping bag temperature (-10*C)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sleeping bag temperature (-15*C)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sleeping bag temperature (-20*C)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tent capacity (1)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tent capacity (2)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tent capacity (3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tent capacity (4)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tent capacity (5)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Size (1L)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Size (2L)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Size (3L)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Properties</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>comboBox_3</receiver>
   <slot>setEditText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Form</receiver>
   <slot>addequipmenttodb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changekgset(QString)</slot>
  <slot>addequipmenttodb()</slot>
 </slots>
</ui>
